gcloud-java-contrib/README.md

```diff
@@ -7,6 +7,12 @@ Contents
 --------
 
  * [gcloud-java-nio](./gcloud-java-nio/): NIO Filesystem Provider for Google Cloud Storage.
+ * [gcloud-java-nio-example](./gcloud-java-nio-example/): How to add GCS NIO after the fact.
```

Comment:
I think using "How to add GCS NIO to existing code" or "legacy code" is a bit more clear than "after the fact".